utils/api.go

```diff
@@ -45,8 +45,7 @@ func buildAPIURL(host string, endpoint string, threeScale string) url.URL {
 	if err != nil {
 		log.Fatal().Err(err).Msgf("Unable to Parse Host URL")
 	}
-	apiURL := url.URL{Host: APIHost.Hostname(), Path: endpoint}
-	apiURL.Scheme = "https"
+	apiURL := url.URL{Host: APIHost.Host, Path: endpoint, Scheme: APIHost.Scheme}
```

Comment:
Can you just override the Path instead of assigning one by one?
```suggestion
	APIHost.Path = endpoint
```

Reply:
Ack. Thanks